### ISSUE TITLE
BUG Encoding issues

### ISSUE THREAD

**Issue by janden** [kymatio/kymatio#294]
I'm getting encoding issues when trying to build the docs (on my laptop [Ubuntu], but not on my desktop [CentOS]). Looks like the reason is that someone has accents in their name...

One solution would be to explicitly define the source files to be UTF-8 (which they are, from what I understand). Another would be to remove the accents.

**Comment by lostanlen** [comment]
Right, I think that the necessary header is


 ```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```

**Comment by janden** [comment]
Yes, but for one (two?) stinking accents in two files (`scattering1d.py` and `setup.py`)? For `setup.py`, sure, since that exports to PyPI, but for the rest, it seems overkill.

I suggest removing accents in `scattering1d.py` and adding the encoding to `setup.py`.

**Comment by janden** [comment]
To clarify, "overkill" because if we do it to `scattering1d.py`, we should be consistent and have it in all source files. `setup.py` is "different" enough that we can have a different format for it.
